.github/actions/microrepo/version-non-npm/README.md

```diff
@@ -0,0 +1,29 @@
+# Micro Repo - Version Non NPM
```

Comment:
I wonder if this should be called something other than “version-non-npm”.  Maybe this one is just called ”version” and then the npm composite action version would be “version-npm”?